PHPCompatibility/Tests/Classes/NewClassesUnitTest.php

```diff
@@ -334,4 +334,15 @@ public function testNoViolationsInFileOnValidVersion()
         $file = $this->sniffFile(__FILE__, '99.0'); // High version beyond newest addition.
         $this->assertNoViolation($file);
     }
+
+    /**
+     * If classes with same name are used in other namespaces, they should not be flagged.
+     *
+     * @return void
+     */
+    public function testNoViolationsInFileIfOtherNamespace()
+    {
+        $file = $this->sniffFile(__DIR__ . '/NewClassesUsesUnitTest.inc', '5.1');
```

Comment:
```suggestion
        $file = $this->sniffFile(__DIR__ . '/NewClassesUsesUnitTest.inc', '4.4');
```

The oldest classes which the sniff can find were introduced in PHP 5.0 and those would now not be tested correctly, so the `testVersion` needs to be below `5.0`.